lib/ansible/module_utils/linode.py

```diff
@@ -4,7 +4,7 @@
 # still belong to the author of the module, and may assign their own license
 # to the complete work.
 #
-# Copyright (c), Luke Murphy @lwm
+# Copyright (c), Luke Murphy @decentral1se
```

Comment:
You may want to change it here as well - https://github.com/ansible/ansible/blob/3ea8e0a14417168be7ada4ed73e6dbc7bb3e4782/.github/BOTMETA.yml#L1583